app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/SurveyQuestion.kt

```diff
@@ -12,19 +12,27 @@ data class SurveyQuestion(
     var answers: List<SurveyAnswer>
 )
 
-enum class QuestionDisplayType(val typeValue: String) {
-    NONE("none"),
-    INTRO("intro"),
-    DROPDOWN("dropdown"),
-    SMILEY("smiley")
+enum class QuestionDisplayType(val typeValue: String, val isIdOnlyAnswer: Boolean) {
+    NONE("none", false),
+    INTRO("intro", false),
+    DROPDOWN("dropdown", true),
+    SMILEY("smiley", true)
```

Comment:
Then we wouldn't need this change 🤔 
```suggestion
enum class QuestionDisplayType(val typeValue: String) {
    NONE("none"),
    INTRO("intro"),
    DROPDOWN("dropdown"),
    SMILEY("smiley")
```